data/data/alibabacloud/slb_attach_controlplane/main.tf

```diff
@@ -0,0 +1,19 @@
+provider "alicloud" {
+  access_key = var.ali_access_key
+  secret_key = var.ali_secret_key
+  region     = var.ali_region_id
+}
+
+locals {
+  cids = "${concat(var.master_ecs_ids, [var.bootstrap_ecs_id])}"
+}
+
+resource "alicloud_slb_backend_server" "slb_attach_controlplane" {
```

Comment:
Maybe a virtual server group [alicloud_slb_server_group](https://registry.terraform.io/providers/aliyun/alicloud/latest/docs/resources/slb_server_group) can be used, of course, this is not required